### PR TITLE
Added build script for azure-storage-blob-go

### PR DESCRIPTION
**Pull request by nailushapotnuru**
## CheckList
Package Name: azure-stoarge-blob-go
Package Version: v0.10.0
Package URL: https://github.com/Azure/azure-storage-blob-go.git

Tested on: UBI 8.5

### CODE REVIEW

File: g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : azure-storage-blob-go
+# Version               : v0.10.0
+# Source repo           : https://github.com/Azure/azure-storage-blob-go.git
+# Tested on             : UBI 8.5
+# Language              : GO
+# Travis-Check          : False
```
Comment (seth-priya) [inline g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh:8]: why is this set to false @nailushapotnuru ?

File: g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+# Disclaimer            : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
```
Comment (seth-priya) [inline g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh:21]: please use "set -e"

File: g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+if [ -z "$1" ]; then
+  export VERSION={1:-v0.10.0}
```
Comment (seth-priya) [inline g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh:23]: there is actually a failure on travis at this line https://app.travis-ci.com/github/ppc64le/build-scripts/builds/244659037, please check and fix

File: g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh (hunk 1 above)
Comment (nailushapotnuru) [reply g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh]: corrected, able to execute the script on ubi container.

File: g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh (hunk 3 above)
Comment (nailushapotnuru) [reply g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh]: done

File: g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh (hunk 2 above)
Comment (nailushapotnuru) [reply g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh]: done

**Review by seth-priya — APPROVED** [review]

**Pull request merged**